core/src/main/java/com/google/googlejavaformat/java/Main.java

```diff
@@ -186,6 +186,17 @@ private int formatFiles(CommandLineOptions parameters, JavaFormatterOptions opti
         outWriter.write(formatted);
       }
     }
+
+    // #846 Clean up any resources and threads created by the executorService.
+    int incompleteFormattingTasks = executorService.shutdownNow().size();
```

Comment:
Let's use [`MoreExecutors.shutdownAndAwaitTermination`](https://guava.dev/releases/snapshot-jre/api/docs/com/google/common/util/concurrent/MoreExecutors.html#shutdownAndAwaitTermination(java.util.concurrent.ExecutorService,java.time.Duration)) here

Reply:
Done although this is a weird use-case for that method since it always requires a timeout. I choose 100ms because the method doesn't allow a timeout of 0. But at this point in the code we have already waited for each task to complete individually in the loop above.